docs/error-codes.md

```diff
@@ -1,7 +1,1593 @@
-# Starcounter error codes
+<!--
+ * THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT.
+ * Source: G:\GitHub\Starcounter\Starcounter.ErrorCodes\src\errorcodes.xml
+ * Generated: 2019-12-11 10:47:48+01:00
+-->
+
+# Starcounter Error Codes
 
 This article lists the error codes that Starcounter uses, along with their severity and descriptions of what they mean.
```

Comment:
```suggestion
This article lists the error codes that Starcounter uses, along with descriptions of what they mean.
```

Reply:
@erikvk  we shall not updated an auto-generated file. All the grammar fixes shall come to the [`Starcounter.ErrorCodes`](https://github.com/Starcounter/Starcounter.ErrorCodes) repository first.

Reply:
ok. never mind then :)